packages/test-app-frontend/src/environment.ts

```diff
@@ -10,5 +10,20 @@ export function applyUrlPrefix(base: string, url = ""): string {
   return normalizedUrl.toString();
 }
 
+/**
+ * Prepends URL hostname with urlPrefix.
+ * dev will default to qa.
+ * no url prefix will default to qa
```

Comment:
Isn't this comment lying? It seems empty VITE_URL_PREFIX will result in base url being unaltered.